runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsPlugin.java

```diff
@@ -45,6 +52,18 @@ public class TileIndicatorsPlugin extends Plugin
 	@Inject
 	private TileIndicatorsOverlay overlay;
 
+	@Inject
+	private Client client;
+
+	@Getter
+	private WorldPoint lastPlayerPosition = null;
+
+	@Getter
+	private int lastTickPlayerMoved = 0;
+
+	@Getter
+	private long lastTimePlayerStoppedMoving = 0;
```

Comment:
long fields default to 0 already
```suggestion
	private long lastTimePlayerStoppedMoving;
```